premise/utils.py

```diff
@@ -16,16 +16,12 @@
 from bw2io.importers.base_lci import LCIImporter
 from country_converter import CountryConverter
 from prettytable import ALL, PrettyTable
-from wurst.linking import (
-    change_db_name,
-    check_duplicate_codes,
-    check_internal_linking,
-    link_internal,
-)
+from wurst.linking import change_db_name, check_internal_linking, link_internal
```

Comment:
which is the right black formatting ?

Reply:
The CI does black auto-formatting, so we can't make this look any different in any case.